v2/apiserver/internal/core/rest/events_endpoints.go

```diff
@@ -213,3 +235,41 @@ func (e *EventsEndpoints) deleteMany(w http.ResponseWriter, r *http.Request) {
 		},
 	)
 }
+
+func eventsSelectorFromURLQuery(
```

Comment:
Should we add unit test coverage for this helper?  (Or covered implicitly by others?)

Reply:
Good question. Not covered implicitly by other tests because the endpoints are the current weak point in our coverage.

I will add a test for this!

Reply:
Tests look great; thank you!